proxy/envoy/policy.go

```diff
@@ -61,6 +61,7 @@ func insertMixerFilter(listeners []*Listener, instances []*model.ServiceInstance
 							"source.ip":  context.IPAddress,
 							"source.uid": context.UID,
 						},
+						QuotaName: "RequestCount",
```

Comment:
The quota name should be part of the proxy config and it is a property of the route.
It also needs to be a `[]string`, because multiple quotas can apply for an incoming request. 

In absence of selector evaluation on the proxy, the proxy *will* call mixer with a quota request for every call. That part is similar to a check request, it is always called.
This argues for selectively enabling mixer calls from the proxy. For example if we know that mixer config does not have any "check" aspects configured, proxy should skip the call.